platform/state/platform_state.proto

```diff
@@ -0,0 +1,391 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+option java_package = "com.hedera.hapi.platform.state.legacy";
+// <<<pbj.java_package = "com.hedera.hapi.platform.state">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+import "basic_types.proto";
+import "timestamp.proto";
+
+/**
+ * The current state of platform consensus.<br/>
+ * This message stores the current consensus data for the platform
+ * in network state.
+ *
+ * The platform state SHALL represent the latest round's consensus.<br/>
+ * This data SHALL be used to ensure consistency and provide critical data for
+ * restart and reconnect.
+ */
+message PlatformState {
+    /**
+     * An address book for this round.
+     * <p>
+     * This SHALL be the latest network-consensus view of consensus nodes.
+     */
+    AddressBook address_book = 1;
+
+    /**
+     * A previous address book.
+     * <p>
+     * If present, the previous address book SHALL be the address book from the
+     * most recent preceding round.
+     * <p>
+     * This is a temporary workaround until dynamic address books are supported.
+     */
+    AddressBook previous_address_book = 2;
+
+    /**
+     * A consensus round.<br/>
+     * The round represented by this state.
+     * <p>
+     * This message SHALL represent the network state after handling of all
+     * transactions reaching consensus in all previous rounds.<br/>
+     * The first state (genesis state) SHALL have a round of 0.<br/>
+     * The first round to handle transactions is defined as round `1`.
+     */
+    uint64 round = 3;
+
+    /**
+     * A running event hash.<br/>
+     * This is computed by the consensus event stream.
+     * <p>
+     * This will be _deprecated_ once the consensus event stream is retired.
+     */
+    bytes legacy_running_event_hash = 4;
+
+    /**
+     * A consensus timestamp for this state.
+     * <p>
+     * This SHALL be the consensus timestamp of the first transaction in
+     * the current round.
+     */
+    proto.Timestamp consensus_timestamp = 5;
+
+    /**
+     * A version describing the current version of application software.
+     * <p>
+     * This SHALL be the software version that created this state.
+     */
+    SoftwareVersion creation_software_version = 6;
+
+    /**
+     * A SHA-384 hash that is the "epoch" hash of this state.
+     * <p>
+     * This SHALL be updated whenever emergency recovery is performed.
+     */
+    bytes epoch_hash = 7;
+
+    /**
+     * A SHA-384 hash that will be the next "epoch" hash.
+     * <p>
+     * This SHALL be used to update the epoch hash at the next round boundary.<br/>
+     * This SHALL NOT be part of the hash and SHALL NOT be serialized.
+     */
+    bytes next_epoch_hash = 8;
```

Comment:
These should be removed, this is not something we support any more.

Reply:
I removed both epoch hash fields.